scripts/docker-entrypoint.sh

```diff
@@ -126,6 +129,25 @@ else
    export SECRET_CASSANDRA_PASSWORD=${CASSANDRA_PASSWORD}
 fi
 
+if [ -e "$hazelcast_username_file" ]; then
+   export SECRET_HZ_CS_AUTH_USERNAME=$(<${hazelcast_username_file})
+else
+   export SECRET_HZ_CS_AUTH_USERNAME=${HZ_CS_AUTH_USERNAME}
+fi
+
+if [ -e "$hazelcast_password_file" ]; then
+   export SECRET_HZ_CS_AUTH_PASSWORD=$(<${hazelcast_password_file})
+else
+   export SECRET_HZ_CS_AUTH_PASSWORD=${HZ_CS_AUTH_PASSWORD}
+fi
+
+if [ "HZ_CLIENT_MODE" == true ]; then
+    if [ "$SECRET_HZ_CS_AUTH_USERNAME" == "" ] || [ "$SECRET_HZ_CS_AUTH_PASSWORD" == "" ]; then
+        echo "HZ_CS_AUTH_USERNAME & HZ_CS_AUTH_PASSWORD must be specified in hazelcast client server mode deployments.";
+        exit 1
+    fi
+fi
+
```

Comment:
Can you add test cases to the changes made around entrypoint. Please refer https://github.com/pegasystems/docker-pega-web-ready/blob/master/tests/pega-web-ready-testcases.yaml to write tests for environment variables and validating mounted secrets.

Reply:
@MadhuriArugula added the test cases